common/web/types/src/kmx/kmx-plus.ts

```diff
@@ -513,6 +514,18 @@ export class List extends Section {
     }
     return this.allocList(strs, s.split(' ').map(unescapeString));
   }
+  /** perform string variable, marker, and unescaping */
+  allocListFromSubstitutedSpaces(s: string, sections: DependencySections): ListItem {
+    if(s === undefined || s === null) {
+      s = '';
+    }
+    return this.allocList(sections.strs, s.split(' ').map(s => {
+      s = sections.vars.substituteStrings(s, sections);
+      s = sections.vars.substituteMarkerString(s);
+      s = unescapeString(s);
+      return s;
+    }));
```

Comment:
```suggestion
    return this.allocList(sections.strs, s.split(' ').map(t => {
      t = sections.vars.substituteStrings(t, sections);
      t = sections.vars.substituteMarkerString(t);
      t = unescapeString(t);
      return t;
    }));
```

Reusing `s` on inner scope is not ideal for readability